pyscriptjs/src/components/pyrepl.ts

```diff
@@ -216,10 +219,15 @@ export class PyRepl extends BaseEvalElement {
     }
 
     getSourceFromElement(): string {
+        if(!this.appendOutput) {
+            this.outputElement.innerHTML = '';
+        }
+
```

Comment:
Thisis a minor comment... 

Not sure about this here. Feels out of the scope of this function, hence hiding behaviour that would be expected elsewhere.

Reply:
That's a good point, I will think about where this could be placed better.

Reply:
I remember, why I placed it there. It's the only function I had available from within the PyRepl component, that gets executed before the source gets evaluated.
As we have postEvaluate, maybe we should add preEvaluate to the base component?

Reply:
Ah! Good point. Think that makes sense, yes :)  TY!

Overall, given the young codebase, there are many small things that we'll need to add to solidify the design.

Reply:
Added two commits to resolve this. Thanks!